app/src/main/java/com/usepace/android/messagingcenter/screens/sendbird/SendBirdChatAdapter.java

```diff
@@ -12,6 +12,7 @@
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.RelativeLayout;
+import android.widget.SeekBar;
 import android.widget.TextView;
 import com.bumptech.glide.Glide;
 import com.dinuscxj.progressbar.CircleProgressBar;
```

Comment:
see if we can use a more active library , it is fine but there are not much contributors to it. and not active as well

Reply:
Sorry, can you explain more ?

Reply:
There is no problem with this library. I just want to say that library is not that popular if we use any more popular one, that being activity updated that will be good.

Reply:
ohh you are talking about CircleProgressBar ?? 
Ok the idea is we build message center on top of send bird sample app, this library was already there